src/main/java/graphql/execution/instrumentation/adapters/ExecutionResultInstrumentationContextAdapter.java

```diff
@@ -1,34 +0,0 @@
-package graphql.execution.instrumentation.adapters;
-
-import graphql.ExecutionResult;
-import graphql.ExecutionResultImpl;
-import graphql.Internal;
-import graphql.execution.instrumentation.InstrumentationContext;
-
-/**
- * A class to help adapt old {@link ExecutionResult} based InstrumentationContext
- * from the newer {@link Object} based ones.
- */
-@Internal
-public class ExecutionResultInstrumentationContextAdapter implements InstrumentationContext<Object> {
-
-    private final InstrumentationContext<ExecutionResult> delegate;
-
-    public ExecutionResultInstrumentationContextAdapter(InstrumentationContext<ExecutionResult> delegate) {
-        this.delegate = delegate;
-    }
-
-    @Override
-    public void onDispatched() {
-        delegate.onDispatched();
-    }
-
-    @Override
-    public void onCompleted(Object result, Throwable t) {
-        if (t != null) {
-            delegate.onCompleted(null, t);
-        } else {
-            delegate.onCompleted(ExecutionResultImpl.newExecutionResult().data(result).build(), null);
```

Comment:
more object creation here via adapter backwards compat.  With this gone - we are more memory performant